src/components/SyrupCard/SyrupCard.tsx

```diff
@@ -101,15 +97,12 @@ const SyrupCard: React.FC<{ syrup: SyrupInfo; dQUICKAPY: string }> = ({
                 <Box display='flex' mt={0.25}>
                   <Typography variant='caption'>
                     $
-                    {Number(
-                      (
-                        syrup.rate * Number(syrup.rewardTokenPriceinUSD)
-                      ).toFixed(2),
-                    ).toLocaleString()}{' '}
-                    <span style={{ color: palette.text.secondary }}>
-                      {' '}
-                      / day
-                    </span>
+                    {syrup.rewardTokenPriceinUSD
+                      ? (
+                          syrup.rate * Number(syrup.rewardTokenPriceinUSD)
```

Comment:
Probably not in this PR but in the future we should figure out a way to remove casting to numbers all the time